src/homebase/datalog_console.cljs

```diff
@@ -0,0 +1,18 @@
+(ns homebase.datalog-console
+  (:require [goog.object :as gobj]
+            [cljs.reader]))
+
+(js/document.documentElement.setAttribute "__datalog-console-remote-installed__" true)
```

Comment:
@groundedSAGE it looks like this breaks Next.js support for HBR with the following error: 
```
error - ReferenceError: document is not defined
    at /Users/jb/repos/supabase-js/example/next-todo/node_modules/homebase-react/dist/js/homebase.react.js:1491:479
    at /Users/jb/repos/supabase-js/example/next-todo/node_modules/homebase-react/dist/js/homebase.react.js:6:22
    at Object.<anonymous> (/Users/jb/repos/supabase-js/example/next-todo/node_modules/homebase-react/dist/js/homebase.react.js:10:3)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at eval (webpack-internal:///homebase-react:1:18)
Installing CLJS DevTools 1.0.2 and enabling features :formatters :hints :async
ReferenceError: document is not defined
    at /Users/jb/repos/supabase-js/example/next-todo/node_modules/homebase-react/dist/js/homebase.react.js:1491:479
    at /Users/jb/repos/supabase-js/example/next-todo/node_modules/homebase-react/dist/js/homebase.react.js:6:22
    at Object.<anonymous> (/Users/jb/repos/supabase-js/example/next-todo/node_modules/homebase-react/dist/js/homebase.react.js:10:3)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at eval (webpack-internal:///homebase-react:1:18)
CLJS DevTools: some custom formatters were not rendered.
https://github.com/binaryage/cljs-devtools/blob/master/docs/faq.md#why-some-custom-formatters-were-not-rendered
CLJS DevTools: some custom formatters were not rendered.
https://github.com/binaryage/cljs-devtools/blob/master/docs/faq.md#why-some-custom-formatters-were-not-rendered
```
Can you please do error catching here and make it a warning when it fails vs breaking compilation?